### PR TITLE
2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps clinical entities and concepts to ICD-10-CM codes using sentence bert embeddings. In this model, synonyms having low cosine similarity to unnormalized terms are dropped. It also returns the official resolution text within the brackets inside the metadata. The model is augmented with synonyms, and previous augmentations are flexed according to cosine distances to unnormalized terms (ground truths).
+
+Outputs 7-digit billable ICD codes. In the result, look for aux_label parameter in the metadata to get HCC status. The HCC status can be divided to get further information: billable status, hcc status, and hcc score.For example, in the example shared below the billable status is 1, hcc status is 1, and hcc score is 11.
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md:23]: lets share Hierarchical Condition Categories (HCC) when we mention for the first time.

File: docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
...out perforation or gangrene]:::mastoiditis in infec/parastc diseases classd elswhr, r ear [mastoiditis in infectious and parasitic diseases classified elsewhere, right ear]:::alcohol induced acute pancreatitis with infected necrosis [alcohol induced acute pancreatitis with infected necrosis]:::oth transient cerebral ischemic attacks and related synd [other transient cerebral ischemic attacks and related syndromes]:::cytomegaloviral pancreatitis [cytomegaloviral pancreatitis]:::ac appendicitis w perf, loc peritonitis, & gangr, w/o abscs [acute appendicitis with perforation, localized peritonitis, and gangrene, without abscess]:::postmenopausal atrophic vaginitis [postmenopausal atrophic vaginitis]:::meningococcal retrobulbar neuritis [meningococcal retrobulbar neuritis]                            | ['1', '0', '1', '0', '1', '1', '1', '1', '1', '0', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1']      | ['0', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '1', '0', '0', '0']      | ['0', '0', '0', '0', '34', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '6', '0', '0', '0']             |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md:151]: replace this chunk with a better one or remove it

File: docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
...::primary inadequate contractions [primary inadequate contractions]:::incomplete defecation [incomplete defecation]:::exhaustion due to excessive exertion, sequela [exhaustion due to excessive exertion, sequela]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | ['0', '0', '0', '1', '0', '1', '1', '1', '1', '1', '0', '1', '1', '1', '1', '1', '1', '1', '1', '0', '1', '1', '1', '1', '1'] | ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0'] | ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0']         |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md:157]: update/remove this chunk

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+| T2DM                                  | PROBLEM  | Q06.0        | Q06.0:::N94.0:::B48.3:::L02.42:::R14.2:::L02.43:::R29.702:::H53.55...                  | amyelia [amyelia]:::mittelschmerz [mittelschmerz]:::geotrichosis [geotrichosis]:::furuncle of limb [furuncle of limb]:::eructation [eructation]:::carbuncle of limb [carbuncle of limb]:::nihss score 2 [nihss score 2]:::tritanomaly [tritanomaly]:::beriberi [beriberi]:::megaloureter [megaloureter]...                                                                                                                                                                                                       | '1||1||72', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '0||0||0', '0||0||0', '1||1||86', '1||0||0', '1||0||0', '1||0||0', '1||0||0'        |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md:155]: remove this chunk

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+| HTG-induced pancreatitis              | PROBLEM  | K85.3        | K85.3:::K85:::K86.0:::K85.2:::K85.1:::K85.0:::K85.32:::K85.8:::B25.2...                | drug induced acute pancreatitis [drug induced acute pancreatitis]:::acute pancreatitis [acute pancreatitis]:::alcohol-induced chronic pancreatitis [alcohol-induced chronic pancreatitis]:::alcohol induced acute pancreatitis [alcohol induced acute pancreatitis]:::biliary acute pancreatitis [biliary acute pancreatitis]:::idiopathic acute pancreatitis [idiopathic acute pancreatitis]...                                                                                                                 | '0||0||0', '0||0||0', '1||1||34', '0||0||0', '0||0||0', '0||0||0', '1||0||0', '0||0||0', '1||1||6', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '1||0||0', '1||1||34', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0'        |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md:156]: remove/change this chunk with `drug-induced pancreatitis`

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+| poor appetite                         | PROBLEM  | R45.81       | R45.81:::R68.82:::Z59.4:::R29.810:::R46.4:::R53.1:::R06.02:::R45.0:::E63.9...          | low self-esteem [low self-esteem]:::decreased libido [decreased libido]:::lack of adequate food [lack of adequate food]:::facial weakness [facial weakness]:::slowness and poor responsiveness [slowness and poor responsiveness]:::weakness [weakness]:::shortness of breath [shortness of breath]:::nervousness [nervousness]:::nutritional deficiency, unspecified [nutritional deficiency, unspecified]...                                                                                                   | '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0'          |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md:161]: remove/update this chunk

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+| poor appetite                         | PROBLEM  | R45.81       | R45.81:::R68.82:::Z59.4:::R29.810:::R46.4:::R53.1:::R06.02:::R45.0:::E63.9...          | low self-esteem [low self-esteem]:::decreased libido [decreased libido]:::lack of adequate food [lack of adequate food]:::facial weakness [facial weakness]:::slowness and poor responsiveness [slowness and poor responsiveness]:::weakness [weakness]:::shortness of breath [shortness of breath]:::nervousness [nervousness]:::nutritional deficiency, unspecified [nutritional deficiency, unspecified]...                                                                                                   |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md:161]: update/remove this chunk

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted clinical entities to ICD-10-CM codes using `sbiobert_base_cased_mli` sentence bert embeddings. In this model, synonyms having low cosine similarity to unnormalized terms are dropped. It returns the official resolution text within the brackets and also provides billable and HCC information of the codes in `all_k_aux_labels` parameter in the metadata. This column can be divided to get further details: `billable status || hcc status || hcc score`. For example, if `all_k_aux_labels` is like `1||1||19` which means the `billable status` is 1, `hcc status` is 1, and `hcc score` is 19.
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md:21]: lets share Hierarchical Condition Categories (HCC) when we mention for the first time.

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+| T2DM                                  | PROBLEM  | Q06.0        | Q06.0:::N94.0:::B48.3:::L02.42:::R14.2:::L02.43:::R29.702:::H53.55...                  | amyelia [amyelia]:::mittelschmerz [mittelschmerz]:::geotrichosis [geotrichosis]:::furuncle of limb [furuncle of limb]:::eructation [eructation]:::carbuncle of limb [carbuncle of limb]:::nihss score 2 [nihss score 2]:::tritanomaly [tritanomaly]:::beriberi [beriberi]:::megaloureter [megaloureter]...                                                                                                                                                                                                       | '1||1||72', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '0||0||0', '0||0||0', '1||1||86', '1||0||0', '1||0||0', '1||0||0', '1||0||0'        |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md:155]: update/remove this chunk

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

**Review by muhammetsnts — APPROVED** [review]

File: docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md (hunk 1 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md:23]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted medical entities to ICD-10-CM codes using `sbiobert_base_cased_mli` Sentence Bert Embeddings and it supports 7-digit codes with HCC status. It has been updated by dropping the invalid codes that exist in the previous versions. In the result, look for the `all_k_aux_labels` parameter in the metadata to get HCC status. The HCC status can be divided to get further information: `billable status`, `hcc status`, and `hcc score`. For example if the result is `1||1||8`: `the billable status is 1`, `hcc status is 1`, and `hcc score is 8`.
```
[the remark this answers is not in the document]
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md (hunk 4 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md (hunk 5 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md (hunk 6 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+| T2DM                                  | PROBLEM  | Q06.0        | Q06.0:::N94.0:::B48.3:::L02.42:::R14.2:::L02.43:::R29.702:::H53.55...                  | amyelia [amyelia]:::mittelschmerz [mittelschmerz]:::geotrichosis [geotrichosis]:::furuncle of limb [furuncle of limb]:::eructation [eructation]:::carbuncle of limb [carbuncle of limb]:::nihss score 2 [nihss score 2]:::tritanomaly [tritanomaly]:::beriberi [beriberi]:::megaloureter [megaloureter]...                                                                                                                                                                                                       |
```
[the remark this answers is not in the document]
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+| HTG-induced pancreatitis              | PROBLEM  | K85.3        | K85.3:::K85:::K86.0:::K85.2:::K85.1:::K85.0:::K85.32:::K85.8:::B25.2...                | drug induced acute pancreatitis [drug induced acute pancreatitis]:::acute pancreatitis [acute pancreatitis]:::alcohol-induced chronic pancreatitis [alcohol-induced chronic pancreatitis]:::alcohol induced acute pancreatitis [alcohol induced acute pancreatitis]:::biliary acute pancreatitis [biliary acute pancreatitis]:::idiopathic acute pancreatitis [idiopathic acute pancreatitis]...                                                                                                                 |
```
[the remark this answers is not in the document]
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md (hunk 7 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md (hunk 8 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md (hunk 9 above)
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+| HTG-induced pancreatitis              | PROBLEM  | K85.3        | K85.3:::K85:::K86.0:::K85.2:::K85.1:::K85.0:::K85.32:::K85.8:::B25.2...                | drug induced acute pancreatitis [drug induced acute pancreatitis]:::acute pancreatitis [acute pancreatitis]:::alcohol-induced chronic pancreatitis [alcohol-induced chronic pancreatitis]:::alcohol induced acute pancreatitis [alcohol induced acute pancreatitis]:::biliary acute pancreatitis [biliary acute pancreatitis]:::idiopathic acute pancreatitis [idiopathic acute pancreatitis]...                                                                                                                 | '0||0||0', '0||0||0', '1||1||34', '0||0||0', '0||0||0', '0||0||0', '1||0||0', '0||0||0', '1||1||6', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '1||0||0', '1||1||34', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0'        |
```
[the remark this answers is not in the document]
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md]: Done!

File: docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+| poor appetite                         | PROBLEM  | R45.81       | R45.81:::R68.82:::Z59.4:::R29.810:::R46.4:::R53.1:::R06.02:::R45.0:::E63.9...          | low self-esteem [low self-esteem]:::decreased libido [decreased libido]:::lack of adequate food [lack of adequate food]:::facial weakness [facial weakness]:::slowness and poor responsiveness [slowness and poor responsiveness]:::weakness [weakness]:::shortness of breath [shortness of breath]:::nervousness [nervousness]:::nutritional deficiency, unspecified [nutritional deficiency, unspecified]...                                                                                                   | '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0'          |
```
[the remark this answers is not in the document]
Comment (Ahmetemintek) [reply docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md]: Done!

**Pull request merged**